server-reboot/server.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/sh
```

Comment:
Same counts for this one, is it different from [common-scripts/basic-gluster.sh](https://github.com/nfs-ganesha/ci-tests/blob/centos-ci/common-scripts/basic-gluster.sh)?

Reply:
Same with https://github.com/nfs-ganesha/ci-tests/blob/centos-ci/common-scripts/basic-gluster.sh

Reply:
Yeah, all scripts should have a #! for the interpreter (/bin/sh in this case, since they're sh scripts)